fuse/main.c

```diff
@@ -21,7 +21,9 @@
 */
 
 #include <exfat.h>
+#ifndef FUSE_USE_VERSION
```

Comment:
I pass the -DFUSE_FUSE_VERSION as part of my Android.mk to let users of other FUSE versions to not hardcode it and give them flexibility, but maybe there's a more flexible way...